firmware/quintuna/VC/src/app/states/app_bmsOnState.c

```diff
@@ -0,0 +1,18 @@
+#include "app_allStates.h"
+#include "app_bmsOnState.h"
+
+static void bmsOnStateRunOnEntry(void) {}
+static void bmsOnStateRunOnTick1Hz(void) {}
+static void bmsOnStateRunOnTick100Hz(void) {}
+static void bmsOnStateRunOnExit(void) {}
+
+const State *app_bmsOnStateState_get(void)
+{
+    static State bmsOn_state = { .name              = "BMS ON",
+                                 .run_on_entry      = bmsOnStateRunOnEntry,
+                                 .run_on_tick_1Hz   = bmsOnStateRunOnTick1Hz,
+                                 .run_on_tick_100Hz = bmsOnStateRunOnTick100Hz,
+                                 .run_on_exit       = bmsOnStateRunOnExit };
```

Comment:
can we run on a system where we extern this? i think that would clean a lot of stuff up, and enable compile time optimizations